Tests/Unit/Invoice/Libraries/PeppolUblXmlCreditNoteTest.php

```
@@ -81,7 +81,7 @@ private function minimalParty(): Party
      *     financial: PeppolFinancialData,
      * }
      */
-    private function minimalDocumentGraph(SettingRepository $s): array
+    private function minimalDocumentGraph(SettingRepository $s, bool $isCreditNote = false): array
```

PHPMD (2.15.0)

[error] The method minimalDocumentGraph has a boolean flag argument $isCreditNote, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)